packages/perseus/src/widgets/definition/definition.tsx

```diff
@@ -92,10 +93,10 @@ class Definition extends React.Component<DefinitionProps> implements Widget {
 
 const styles = {
     tooltipBody: {
-        color: color.offBlack,
-        fontSize: 20,
-        fontWeight: 500,
-        lineHeight: "30px",
+        color: semanticColor.core.foreground.neutral.strong,
+        fontSize: "2rem",
+        fontWeight: font.weight.medium,
+        lineHeight: 1.5,
```

Comment:
This also may need a new token, though this value may work as it is. Will also confirm with designer.